src/materials/mat_instanced.ts

```diff
@@ -0,0 +1,79 @@
+import {InstancedAttribute} from "../components/com_render_vox.js";
+import {GL_TRIANGLES} from "../webgl.js";
+import {link, Material} from "./mat_common.js";
+
+let vertex = `#version 300 es\n
+    // Matrices: PV, world, self
+    uniform mat4 p,q,r;
```

Comment:
Please use a non-minified version of the shader. You can refer to https://github.com/piesku/backcountry/blob/bdc30aca7dd15f23da608f8c9ec8f458a5e28d3b/shaders/instanced.vert and https://github.com/piesku/backcountry/blob/0653b8a5001405f1ea382aa700fe3114f457181a/src/materials/mat_instanced.ts.